### ISSUE TITLE
[ENH] Calibration plots for probabilistic predictions

### ISSUE THREAD

**Issue by fkiraly** [sktime/skpro#162]
@benHeid made nice calibration plots here: https://github.com/sktime/sktime/pull/5632

So I wonder:

* should these also be added in `skpro` for probabilistic tabular regressors? Same principle works.
* any reasons why it could not be shifted via copy/paste and import change?

Mid-term, it might be best to have them only in `skpro`, and make `sktime` forecasting rely more on `skpro` distributions machinery and utilities.

FYI @benHeid, any thoughts?

**Comment by benHeid** [comment]
I suppose shifting and changing import should work. 

I agree, that it would be better to have anything probabilistic related that is not specific to time series should be located in skpro.

**Comment by fkiraly** [comment]
> I agree, that it would be better to have anything probabilistic related that is not specific to time series should be located in skpro.

Yes, though I would do this only slowly over time with the interfaces maturing.

**Comment by MBKKHAN** [comment]
Hi @fkiraly, I am interested in working on this for GSoC 2026. I have implemented a `plot_calibration` function following the pattern of existing plotting utilities in `skpro/utils/plotting.py`. The function plots a calibration curve comparing predicted quantile levels to observed coverage fractions.
